lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -270,7 +286,11 @@ def mymove(source: Path, destination: Path, *args, **kwargs) -> Path:
             dest.append(d)
             if source.suffix == allow:
                 return real_move(source, destination, *args, **kwargs)
-            raise Exception("I'm broken")
+            if errno:
+                e = OSError(errno, "something went badly")
+            else:
+                e = Exception("I broke")
+            raise e
```

Comment:
`errno` is never false-y...is the parametrize-ation missing a case, or should this code be simplified?  (Ditto line 309.)

Reply:
Yeah, I'd meant to have a third alternative for each of the two sets. Dang.